### PR TITLE
Changing the HTTP response of readiness probe to 503 when not available

### PR DESCRIPTION
**Pull request by LalatenduMohanty**
Signed-off-by: Lalatendu Mohanty <lmohanty@redhat.com>

### CODE REVIEW

File: graph-builder/src/status.rs [hunk 1]
```diff
@@ -23,12 +23,12 @@ pub async fn serve_liveness(app_data: actix_web::web::Data<State>) -> Fallible<H
 ///
 /// Status:
 ///  * Ready (200 code): a JSON graph as the result of a successful scrape is available.
-///  * Not Ready (500 code): no JSON graph available yet.
+///  * Service Unavailable (503 code): no JSON graph available yet.
```
Comment (lucab) [inline graph-builder/src/status.rs]: The docstring is meant to describe the semantics of the result ("ready" or "not ready"), not the meaning of the HTTP code.

**Review by vrutkovs — CHANGES_REQUESTED** [review]
Needs to update readiness check - and same applied to policy-engine

File: graph-builder/src/status.rs (hunk 1 above)
Comment (steveej) [reply graph-builder/src/status.rs]: To add to Luca's comment:

```suggestion
///  * Not Ready (503 code): no JSON graph available yet.
```

**Comment by LalatenduMohanty** [comment]
> Needs to update readiness check - and same applied to policy-engine

I do not think I understand. Can you please elaborate?

**Comment by vrutkovs** [comment]
>I do not think I understand. Can you please elaborate?

This PR updates only one check - readiness check in graph-builder. It should also update status returned by readiness check there - and update policy-engine binary too

**Review by vrutkovs — APPROVED** [review]

**Pull request merged**